pyanaconda/ui/gui/spokes/subscription.py

```diff
@@ -78,7 +78,7 @@ def should_run(cls, environment, data):
         return is_module_available(SUBSCRIPTION)
 
     def __init__(self, *args):
-        super().__init__()
+        super().__init__(self, *args)
```

Comment:
I don't think that `self` should be there.

Reply:
Fixed.